server/commands/teamUpdater.ts

```diff
@@ -1,9 +1,9 @@
 import { Transaction } from "sequelize";
 import { sequelize } from "@server/database/sequelize";
-import { Event, Team, User } from "@server/models";
+import { Event, Team, TeamDomain, User } from "@server/models";
 
 type TeamUpdaterProps = {
-  params: Partial<Team>;
+  params: Partial<Team> & { allowedDomains: string[] };
```

Comment:
Not sure if this wants to be cleaned up

`allowedDomains` is not a _real_ attribute of a `Team` but its what the front-end sends us. We translate these into the TeamDomain models